tests/behat/features/single-page-routing.feature

```diff
@@ -0,0 +1,15 @@
+@retry @job2
+Feature: Single-page routing works as expected
```

Comment:
Weirdly named file and feature, I don't quite understand what it's supposed to mean

Reply:
Named after "single page apps" which the CMS routing kinda tries to mimic.

Reply:
`model-admin.or-something-like-that.newnamehere.feature` :p
`model-admin` isn't at all what this is for. I'll give it a new name that's as good as the current one and you can tell me to change it again.